lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -150,7 +150,7 @@ def run!(verify_result: true, allow_protected_ns: false, prune: true)
         @logger.phase_heading("Predeploying priority resources")
         start_priority_resource = Time.now.utc
         predeploy_priority_resources(resources)
-        ::StatsD.measure('priority_resources.duration', StatsD.duration(start_priority_resource), tags: statsd_tags)
+        StatsD.measure('priority_resources.duration', StatsD.duration(start_priority_resource), tags: statsd_tags)
```

Comment:
I've got a PR for this as an fyi https://github.com/Shopify/kubernetes-deploy/pull/383